stingray/bispectrum.py

```diff
@@ -78,6 +79,23 @@ def compute_bispectrum(self, lc):
         bispec = scipy.fftpack.fftshift(bispec) / self.k
         return bispec
 
+    def plot(self):
+        bispectrum = self.compute_bispectrum(self.lc)
+        print bispectrum
+        if self.nfft % 2 == 0:
+            freaxis = np.transpose(
+                np.arange(-1 * self.nfft / 2, self.nfft / 2))
+        else:
+            freaxis = np.transpose(
+                np.arange(-1 * (self.nfft - 1) / 2, (self.nfft - 1) / 2 + 1))
+        cont = plt.contourf(freaxis, freaxis, abs(
+                    bispectrum), 200, cmap=plt.cm.Spectral_r)
+        plt.colorbar(cont)
+        plt.title('Bispectrum ')
+        plt.xlabel('f1')
+        plt.ylabel('f2')
+        plt.show()
```

Comment:
I would go with the same decisions we've made for the `draw` and `savefig` methods for `Lightcurve` and `Powerspectrum` here. 
And yes, I think we shouldn't have `pt.show()` in here. In principle, perhaps instead of using the `pyplot` functions, it might be useful to generate a `Figure` object and return that to the user instead?